### ISSUE TITLE
Recent projects Panel does not have a Scrollbar

### ISSUE THREAD

**Issue by stefanhahmann** [mastodon-sc/mastodon#350]
If many projects appear in the Recent Projects Panel of the Mastodon Launcher, it has the effect that the option "Open another project" is not shown anymore. It would be better, if instead a scrollbar appears that allows to scroll trough the list of recent projects while at the same time leaving enough space for the "Open another project" text and related button.

* Enough space

![grafik](https://github.com/user-attachments/assets/a66d2465-9ef5-4578-bc99-c712538ba906)

* Not enough space. "Open another project" disappears:

![grafik](https://github.com/user-attachments/assets/e70043b6-d941-4a88-96c4-0d089f07c3d3)

**Comment by tinevez** [comment]
Vlado told me the same thing. What about instead of a scrollbar we restrict the number of projects shown here to 5? 
And how to make it better looking?

**Comment by stefanhahmann** [comment]
> Vlado told me the same thing. What about instead of a scrollbar we restrict the number of projects shown here to 5?

I only noticed after having finished the solution with the layout change that restricting to 5 projects would also solve the problem. However, I quite like this feature. Reducing to 5 would feel as a step backwards to me. One could restrict to 9 without changing the layout and still solve the problem that the button is not shown, when there are 10.

I just checked:
* Microsoft Word also shows *10* recent projects by default
* IntelliJ Idea shows *18* recent projects by default 
* GIMP shows *10* recent documents by default
* FIJI show *15* recent documents by default

I would thus be in favor to be able to have at least 10 recent projects, maybe even 15. 20 as the PR suggest may however not be required.

> And how to make it better looking?

Hm, I am not an expert in this and this was not part of this PR. I just wanted to fix the inconvenience that the "Open another project" button disappears.
